docs/book/src/design/wallet.md

```diff
@@ -0,0 +1,13 @@
+# Wallet
+
+This provides the functionality of a non-custodial wallet to end-users without smartphones, accessible through a smartphone-based connector app with limited trust.
```

Comment:
This requires some elaboration.  It is not immediately clear how an application can be both "smartphone-based" while also targeting users *without* smartphones.  I assume this is where the "limited trust" comes in but there is value in being explicit here.

Reply:
How about something like this?

```suggestion
This provides the functionality of a non-custodial wallet to end-users who don't own a smartphone themselves, accessible through a smartphone-based connector app with limited trust.
```

Reply:
I think that is much clearer yes!